rust-peer/src/main.rs

```diff
@@ -191,11 +195,14 @@ struct Behaviour {
     identify: identify::Behaviour,
     kademlia: Kademlia<MemoryStore>,
     keep_alive: keep_alive::Behaviour,
-    // ping: ping::Behaviour,
+    ping: ping::Behaviour,
```

Comment:
We don't need ping imo. Was this for debugging?

Reply:
it's necessary if we want to display the [Latency of nearest peer](https://github.com/libp2p/universal-connectivity/pull/39/files#diff-2ddf273f11d6f40730b230314e354e9b900ee9a6190738168c8d8d18b7a369b4R96)

Reply:
I say remove. The latency is interesting but not at the cost the of the inbound connections issue